.bazelci/config.yaml

```diff
@@ -54,6 +54,19 @@ tasks:
       - "//:third_party_examples_linux_tests"
     build_targets: *linux_targets
     test_targets: *linux_targets
+  ubuntu2004_examples_bzlmod:
+    name: Examples (bzlmod)
```

Comment:
No MacOS or windows builds?

Reply:
Most of the issues with bzlmod specific issues happen at analysis time; I initially have just left this scoped to linux just for the sake of keeping CI times down as macOS in particular is slow.